migration-engine/connectors/sql-migration-connector/src/sql_destructive_change_checker/unexecutable_step_check.rs

```diff
@@ -5,33 +5,52 @@ use super::{
 
 #[derive(Debug)]
 pub(crate) enum UnexecutableStepCheck {
-    AddedRequiredFieldToTable { table: String, column: String },
-    AddedRequiredFieldToTableWithPrismaLevelDefault { table: String, column: String },
-    MadeOptionalFieldRequired { table: String, column: String },
-    MadeScalarFieldIntoArrayField { table: String, column: String },
-    DropAndRecreateRequiredColumn { table: String, column: String },
+    AddedRequiredFieldToTable {
+        table: String,
+        column: String,
+        namespace: Option<String>,
+    },
+    AddedRequiredFieldToTableWithPrismaLevelDefault {
+        table: String,
+        column: String,
+        namespace: Option<String>,
+    },
+    MadeOptionalFieldRequired {
+        table: String,
+        column: String,
+        namespace: Option<String>,
+    },
+    MadeScalarFieldIntoArrayField {
+        table: String,
+        column: String,
+        namespace: Option<String>,
+    },
+    DropAndRecreateRequiredColumn {
+        table: String,
+        column: String,
+        namespace: Option<String>,
+    },
```

Comment:
Could the data in these enums be a good candidate for the `Table` and `Column` structs you defined in the previous PR?